.github/pull_request_template.md

```diff
@@ -0,0 +1,51 @@
+## Overview
+<!--  
+Provide a high-level description of this change.   
+-->
+
+
+
+## Type of change
+<!--  
+Check the box below that describes your change best:
+--> 
+
+- [ ] new plugin
+- [ ] improvement for an existing plugin
+- [ ] bug-fix for an existing plugin
+- [ ] changes in contributor's experience
+
+## Related Issue(s)
+<!--  
+If applicable - add the issue that your PR relates to or closes:
+  - use Resolves: #ISSUE_NUMBER to trigger closing of the issue on merge of this PR  
+  - use Relates: #ISSUE_NUMBER to indicate relation to an issue, but issue will not close  
+-->  
+
+* Resolves: #
+* Relates: #
+
+## How To Test
+<!--
+Provide testing instructions for validating the changes introduced in this PR.
+This will serve as a starting point for your reviewers, for functional testing.
+-->
+
+
+
+## Changelog
+<!--  
+A one line sentence describing the change that this PR introduces. 
+If this has impact over the user experience, your changelog will be included in the release notes of the next stable version of 1Password CLI.
+
+Here are a few guidelines for writing a good changelog:
+- Keep your description to a single sentence if you can, and use proper capitalization and punctuation, including a final period.
+- Don't use emoji in your description.
+- Avoid starting your sentence with "improved" or "fixed", especially when using those change types. Instead, describe the improvement or say what you fixed.
+- Avoid using terminology like "Users are shown" or "You can now" and instead focus on the thing that was changed.
+
+Have a look over 1Password CLI's past release notes for examples: 
+https://app-updates.agilebits.com/product_history/CLI2
+-->  
```

Comment:
Would be nice to add a few examples here instead of only linking to the app-updates pages.

Reply:
Done!